### PR TITLE
MOBILEAPPS-1236

### PR DESCRIPTION
**Pull request by aman-alfresco**
- add a framework for the analytics

### CODE REVIEW

**Review by ankitgoyalgloballogic — APPROVED** [review]

File: app/google-services.json [hunk 1]
```diff
@@ -1,37 +1,83 @@
 {
```
Comment (gl-lovekesh) [inline app/google-services.json:1]: This file can not commit right now. Please revert the changes for this file.

Comment (aman-alfresco) [reply app/google-services.json]: Done

**Review by gl-lovekesh — APPROVED** [review]
